doc/classes/LightmapperRD.xml

```diff
@@ -5,7 +5,7 @@
 	</brief_description>
 	<description>
 		LightmapperRD ("RD" stands for [RenderingDevice]) is the built-in GPU-based lightmapper for use with [LightmapGI]. On most dedicated GPUs, it can bake lightmaps much faster than most CPU-based lightmappers. LightmapperRD uses compute shaders to bake lightmaps, so it does not require CUDA or OpenCL libraries to be installed to be usable.
-		[b]Note:[/b] Only usable when using the RenderingDevice backend (Forward+ or Mobile renderers), not Compatibility.
+		[b]Note:[/b] This lightmapper requires the GPU to support the [RenderingDevice] backend (Forward+ or Mobile renderers). When using the Compatibility renderer, baking will use a temporary [RenderingDevice]. [i]Rendering[/i] lightmaps that were already baked beforehand does not require support for RenderingDevice.
```

Comment:
One was changed to a link but the other wasn't. Also, maybe the following flows better?
```suggestion
		[b]Note:[/b] This lightmapper requires the GPU to support the [RenderingDevice] backend (Forward+ or Mobile renderers). When using the Compatibility renderer, baking will use a temporary [RenderingDevice]. Support for [RenderingDevice] is not required to [i]render[/i] lightmaps that were already baked beforehand.
```